markets/perps-market/cannonfile.toml

```diff
@@ -145,6 +148,7 @@ func = "initializeFactory"
 args = [
     "<%= imports.synthetix.contracts.CoreProxy.address %>",
     "<%= imports.spotMarket.contracts.SpotMarketProxy.address %>",
+    "<%= settings.minDelegationTime %>",
```

Comment:
I don't see any changes in contract. Is this necessary?

Reply:
reverted external problematic change after discussion